From c56a21d965827bccb6cf99e8d80a79f39ba5befc Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Wed, 13 Feb 2019 11:13:34 +0800
Subject: [PATCH] arch: arm64: dts: change fpga image file name

Change the FPGA update image file name into system.bit.bin

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/xilinx/zynqmp-zcu102-fpga-update.dts  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-fpga-update.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-fpga-update.dts
index fc64598..05a9feb 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-fpga-update.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-fpga-update.dts
@@ -12,7 +12,7 @@
 			#address-cells = <2>;
 			#size-cells = <2>;
 
-			firmware-name = "Bitstream.bin";
+			firmware-name = "system.bit.bin";
 		};
 	};
 };
-- 
1.7.9.5

